discord/commands/options.py

```diff
@@ -390,6 +387,47 @@ def to_dict(self) -> dict:
     def __repr__(self):
         return f"<discord.commands.{self.__class__.__name__} name={self.name}>"
 
+    @property
+    def autocomplete(self):
```

Comment:
Needs a docstring? Simply moving the ``autocomplete`` attribute docstring from the ``Attributes`` section on the class to here would fix it.